docs/admin_guide.md

```diff
@@ -908,7 +908,7 @@ With the following content:
 
 ```ini
 [program:messenger]
-command=php /var/www/mbin/bin/console messenger:consume async async_ap failed --time-limit=3600
```

Comment:
owo no again an update on the messenger lol lol. Hopefully this will be the last change, if not, add now as many as you want. Creating no further impact on the setup for admins in the future ideally.